modules/ROOT/pages/install-rtfctl.adoc

```diff
@@ -29,6 +29,25 @@ To view a list of all supported commands, run `rtfctl -h`.
 
 To get more information on a specific command, run `rtfctl <command> -h`.
 
+== Usage
+
+* Install a Mule license
+
+.. Make sure to base64 encode the license following the instructions in xref:install-manual#base64-encode-your-mule-license-key[Base64 Encode your Mule License Key].
+.. Run the following command:
++
+----
+$ sudo ./rtfctl apply mule-license '<license-key-information>'
+Updating rtf namespace... OK
+----
+
+* Collect debug package information for an application
```

Comment:
I think this section belongs in another topic -- possibly troubleshooting.

Reply:
Good point - I've moved this. Please check to make sure my intro sentence is correct. Thank you! My big plan is to go through all the content and reorganize it. There are other cases of information needing to be moved IMHO.